index.html

```diff
@@ -0,0 +1,156 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="stylesheet" href="style.css">
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>portfolio</title>
+</head>
+<body>
+  <div class="content">
+  <header class="site-header">
+    <div class="site-wrapper">header</div>
+  </header>
+  <main class="site-main">
+    <section class="first-section">
+      <div class="site-wrapper">
+        <p>text</p>
+      </div>
+    </section>
+    <section class="second-section">
+      <div class="site-wrapper">
+        <header class="second-section-title">header</header>
+        <p>text</p>
+      </div>
+    </section>
+    <section class="third-section">
+      <div class="site-wrapper">
+        <header class="project-list-header">header</header>
+        <ul class="project-list">
+          <li class="project-list-item">item 1
+            <img>
+            <section class="feature1">
+              <header class="title-feature1">header</header>
+              <p class="content-feature1">text</p>
+              <ul class="link-list">
+                <li class="link-list-item">1</li>
+                <li class="link-list-item">2</li>
+                <li class="link-list-item">3</li>
+              </ul>
+          </li>
+          <li class="project-list-item">item 2
+            <img>
```

Comment:
There are couple of empty `<img>` tags here. Could you explain the reason or just remove it?

Reply:
I removed it.